libpod/container_config.go

```diff
@@ -258,7 +258,7 @@ type ContainerNetworkConfig struct {
 	// this is only the set of aliases the container was *created with*.
 	// Formatted as map of network name to aliases. All network names must
 	// be present in the Networks list above.
-	NetworkAliases map[string][]string `json:"network_alises,omitempty"`
+	NetworkAliases map[string][]string `json:"network_aliases,omitempty"`
```

Comment:
@mheon PTAL

I feel like this might break aliases.

Reply:
I'm going to wait to do anything until I see a response (the CI seems fairly expensive).

Reply:
Yeah, this is a breaking change to DB schema - need to leave as-is. Good catch @Luap99

Reply:
Would you take a comment indicating it's a known botch? (I wouldn't include it in this PR, just asking for in general)

Reply:
Sure, that's always welcome

Reply:
Any particular style to it?

Reply:
@mheon / @rhatdan: is there a style I should use to call this out?

Reply:
"The JSON tag is misspelled, but this must be retained for database compatibility" maybe?